### ISSUE TITLE
Hip73 deselection 

### ISSUE THREAD

**Issue by Apowell11** [helium/HIP#496]
Would I  likely become deselected as a poor validator?  I am experiencing multiple install issues with the authorized Freedom Fi 5g miner installers. I have difficulty finding Knowledgeable support. On-Tech blaming technology difficulties on Freedom Fi. freedom Fi hard to reach. Customer support . On-Tech installers are on a steep learning curve  that affects our effectiveness and validation quality.  Are there others like me, that could experience negative impacts if Hip 73 deselection takes affect to deselect quicker? Does passage of Hip73 eliminate the smaller minors? How do I solve this issue for two minors?

**Comment by TheRealJohnMac50** [comment]
You mention FFI 5G small cells and miners, which have nothing to do with validators, nor this HIP.

**Comment by abhay** [comment]
closing. thanks @TheRealJohnMac50

**Comment by Apowell11** [comment]
> You mention FFI 5G small cells and miners, which have nothing to do with validators, nor this HIP.

**Comment by Apowell11** [comment]
Okay. Thank you.

**Comment by TheRealJohnMac50** [comment]
> Okay. Thank you.

You're welcome 🙂👍🏻

**Comment by Apowell11** [comment]
So deselection of poor validators doesn’t affect the small minor? Why are there poor validators? My apologies for my ignorance. I’m trying to learn more

**Comment by TheRealJohnMac50** [comment]
> So deselection of poor validators doesn’t affect the small minor? Why are there poor validators? My apologies for my ignorance. I’m trying to learn more

In layman's terms, this HIP will ensure that poor performing validators (vals) will be more excluded from the CG (Concensus Group). Which will esnure that only better performing vals will be in the CG, that will help your (LoRa) miners better peform PoC (proof of coverage), and ultimately earn more PoC rewards. This kind of discussion may be better discussed in the Helium socials (discord, Tg, exc.) than here.
